### ISSUE TITLE
Add support for double period classes

### ISSUE THREAD

**Issue by tkon99** [lesander/magister-calendar#26]
For example, having maths the 7th and 8th hour results in two separate appointments.
Although I recommend making it optional.

**Comment by lesander** [comment]
Absolutely, we could merge those appointments together. I'll look into it, or do you have time for a PR?

**Comment by tkon99** [comment]
I don't have much time on hand during the next few weeks, so I'd really appreciate if you looked into it.

**Comment by lieuwex** [comment]
I guess checking the if the location and class are the same and the `beginBySchoolhour`s have a difference of 1 will be enough.
